From 3eb064ffa9b43fb5f245b8abda48a1c152a6f2d7 Mon Sep 17 00:00:00 2001
From: me <noemail@nowhere.com>
Date: Thu, 20 Nov 2014 14:37:34 -0700
Subject: [PATCH] Append to CFLAGS and CXXFLAGS as setup by ./configure.  This
 can allow users to Customize the SQLite build process with compile time
 #defines rather than altering the sources.

Signed-off-by: Max Filippov <jcmvbkbc@gmail.com>
---
Backported from: 3eb064ffa9b43fb5f245b8abda48a1c152a6f2d7

 Makefile-shared.mk | 4 ++--
 Makefile-static.mk | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile-shared.mk b/Makefile-shared.mk
index b2a4920..d15ce3c 100644
--- a/Makefile-shared.mk
+++ b/Makefile-shared.mk
@@ -12,10 +12,10 @@ OBJS= \
 	${objsdir}/sqlite3.o
 
 # C Compiler Flags
-CFLAGS= -fPIC -MMD -MP
+CFLAGS+= -fPIC -MMD -MP
 
 # CC Compiler Flags
-CPPFLAGS= -DKOMPEX_SQLITEWRAPPER_EXPORT -DKOMPEX_SQLITEWRAPPER_DYN -fPIC -MMD -MP -I${includedir}
+CPPFLAGS+= -DKOMPEX_SQLITEWRAPPER_EXPORT -DKOMPEX_SQLITEWRAPPER_DYN -fPIC -MMD -MP -I${includedir}
 
 # Link Libraries and Options
 LDLIBSOPTIONS= -shared -fPIC
diff --git a/Makefile-static.mk b/Makefile-static.mk
index 429a4ff..0678681 100644
--- a/Makefile-static.mk
+++ b/Makefile-static.mk
@@ -12,10 +12,10 @@ OBJS= \
 	${objsdir}/sqlite3.o
 
 # C Compiler Flags
-CFLAGS= -MMD -MP
+CFLAGS+= -MMD -MP
 
 # CC Compiler Flags
-CPPFLAGS= -I${includedir} -MMD -MP
+CPPFLAGS+= -I${includedir} -MMD -MP
 
 # Link Libraries and Options
 LDLIBSOPTIONS=
